src/LodeRunner.API/src/Core/SystemConstants.cs

```diff
@@ -66,7 +66,7 @@ public class SystemConstants
         /// <summary>
         /// The error message success.
         /// </summary>
-        public const string ErrorMessageSuccess = "The parameter 'cientStatusId' should be a non-empty string";
+        public const string ErrorMessageSuccess = "The parameter 'cientStatusId' should be a non-empty GUID string";

```

Comment:
Small comment :) - I noticed a minor typo throughout the files here: 'cientStatusId' instead of clientStatusId

Reply:
Nice catch!